packages/squiggle-lang/src/rescript/Distributions/PointSetDist/Continuous.res

```diff
@@ -87,7 +87,6 @@ let stepwiseToLinear = (t: t): t =>
 // Note: This results in a distribution with as many points as the sum of those in t1 and t2.
 let combinePointwise = (
   ~integralSumCachesFn=(_, _) => None,
-  ~integralCachesFn: (t, t) => option<t>=(_, _) => None,
```

Comment:
How sure are you that we don't need this? It's been a while since I studied this part of the codebase

Reply:
Because the compiler said it is not ever used in the function body, and it compiles and runs fine without it. If you did need it, then the warning wouldn't be there and I've just identified a bug in the code